modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/AssignmentStats.java

```diff
@@ -0,0 +1,100 @@
+package com.intuit.wasabi.repository.cassandra.impl;
+
+import com.intuit.wasabi.assignmentobjects.Assignment;
+import com.intuit.wasabi.experimentobjects.Bucket;
+import com.intuit.wasabi.experimentobjects.Experiment;
+import com.intuit.wasabi.repository.cassandra.accessor.count.HourlyBucketCountAccessor;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.*;
```

Comment:
@mooxter Please don't use `.*` imports, as this loads all (MANY 😄 ) the classes under java.util. into the JVM. Please import specific classes instead that you're actually using. There's a shortcut to for this in your IDE (IntelliJ?).